test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -122,6 +122,19 @@ public void AutoFitRegressionTest()
 
         [LightGBMFact]
         public void AutoFitRankingTest()
+        {
+            // Testing Ranking with both TrainValidateRunner and CrossValSummaryRunner
+            // with corresponding baseline metrics. The metrics of Ranking with
+            // CrossValSummaryRunner is lower due to the 10 cross validation folds
+            // and due to the fact that CrossValSummaryRunner is used only when
+            // fewer than <1500 rows of training data is given.
+            AutoFitRankingTestTemplate(true, 0.4, 20);
+            AutoFitRankingTestTemplate(false, 0.13, 9.7);
+            AutoFitRankingCVTestTemplate(true, 0.4, 19);
+            AutoFitRankingCVTestTemplate(false, 0.13, 9.7);
+        }
+
+        private void AutoFitRankingTestTemplate(bool isUsingTrainValidateRunner, double maxNdcg, double maxDcg)
```

Comment:
I would use another names besides `maxDcg` as this a term used within the calculation of the NDCG --  `NDCG = DCG / MaxDCG` ([code](https://github.com/dotnet/machinelearning/blob/d849ba4c7831586018d5b2cca6bdd2b1fa228ddf/src/Microsoft.ML.FastTree/Training/DcgCalculator.cs#L293)).

```suggestion
        private void AutoFitRankingTestTemplate(bool isUsingTrainValidateRunner, double minExpectedNdcg, double minExpectedDcg)
```

Reply:
You may want as "min" instead of "max", since your comparison is checking the metric value is larger than the given. Hence the given value is the minimum allowed value.